prisma/seed.ts

```diff
@@ -3,7 +3,7 @@
 // Seed data for the database
```

Comment:
I was trying to see if Reviewdog would do something here by editing the import line 🤔